invenio_workflows/models.py

```diff
@@ -153,7 +153,7 @@ class WorkflowObjectModel(db.Model):
             postgresql.JSON(none_as_null=True),
             'postgresql',
         ),
-        default={},
```

Comment:
imo yes, though we are using lambda everywhere else, so better keep the style and refactor all at once later (whenever... not a big deal).